### PR TITLE
[sensor.AFRICA] Disable map zoom on embed

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fix issue # https://github.com/CodeForAfrica/sensors.AFRICA/pull/226

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -112,32 +116,24 @@ const tiles = L.tileLayer(config.tiles, {
...
-let coordsCenter = config.center;
-let zoomLevel = config.zoom;
-
-if (location.hash) {
+} else if (location.hash) {
+		// Coordinates are passed by hash path
+	  // /#{zoom}/{lat}/{lng}
 	const hash_params = location.hash.split("/");
 	coordsCenter = [hash_params[1], hash_params[2]];
 	zoomLevel = hash_params[0].substring(1);
```
Comment (DavidLemayian) [inline src/js/index.js:142]: Why are we using our own center and zoom instead of using hashpaths similar to here?

Comment (nerdlet) [reply src/js/index.js:142]: Hi @DavidLemayian We are catering for query instances for when the map is on the site like such https://github.com/CodeForAfrica/sensors.AFRICA/blob/master/src/components/SensorMap/index.js and here as well https://github.com/CodeForAfrica/sensors.AFRICA/blob/master/src/components/Embeds/AirMap.js 
Then again I could have missed something here that we could have used instead, like the hashpaths

File: src/js/index.js [hunk 2]
```diff
@@ -133,7 +136,9 @@ if (query.no_overlay === "false") d3.select("#betterplace").style("display", "in
 let coordsCenter = config.center;
 let zoomLevel = config.zoom;
 
-if (location.hash) {
+ if (location.hash) {
+		// Coordinates are passed by hash path
+	  // /#{zoom}/{lat}/{lng}
```
Comment (DavidLemayian) [inline src/js/index.js]:
```suggestion
	// /#{zoom}/{lat}/{lng}
```

File: src/js/index.js [hunk 3]
```diff
@@ -133,7 +136,9 @@ if (query.no_overlay === "false") d3.select("#betterplace").style("display", "in
 let coordsCenter = config.center;
 let zoomLevel = config.zoom;
 
-if (location.hash) {
+ if (location.hash) {
+		// Coordinates are passed by hash path
```
Comment (DavidLemayian) [inline src/js/index.js]:
```suggestion
	// Coordinates are passed by hash path
```

File: src/js/index.js [hunk 4]
```diff
@@ -102,6 +102,8 @@ const panelIDs = {
 
 const div = d3.select("#sidebar").append("div").attr("id", "table").style("display", "none");
 
+
+
```
Comment (DavidLemayian) [inline src/js/index.js]:
```suggestion
```

File: src/js/index.js [hunk 5]
```diff
@@ -102,6 +102,8 @@ const panelIDs = {
 
 const div = d3.select("#sidebar").append("div").attr("id", "table").style("display", "none");
 
+
```
Comment (DavidLemayian) [inline src/js/index.js]:
```suggestion
```

**Pull request merged**